### PR TITLE
Trev8939/identify layers

### PR DESCRIPTION
**Pull request by TADraeseke**
https://devtopia.esri.com/runtime/java-android-sdk/issues/1390

### CODE REVIEW

**Comment by TADraeseke** [comment]
@doneill. Have scope for a quick review? Thanks!

File: identify-layers/src/main/java/com/esri/arcgisruntime/sample/identifylayers/MainActivity.java [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+   *
+   * @param identifyLayerResults a list of identify results generated in identifyResult().
+   */
+  private void handleIdentifyResults(List<IdentifyLayerResult> identifyLayerResults) {
+    String message = "";
+    int totalCount = 0;
+    for (IdentifyLayerResult identifyLayerResult : identifyLayerResults) {
+      int count = geoElementsCountFromResult(identifyLayerResult);
+      String layerName = identifyLayerResult.getLayerContent().getName();
+      message += layerName + ": " + count;
```
Comment (doneill) [inline identify-layers/src/main/java/com/esri/arcgisruntime/sample/identifylayers/MainActivity.java]: Here is an instance where `StringBuilder` should be used: 

```java
StringBuilder message = new StringBuilder(); 
...
message.append(layerName).append(": ").append(count);
```

**Comment by TADraeseke** [comment]
Thanks @doneill! I've refactored to StringBuilder as per your suggestion.

**Comment by doneill** [comment]
@TADraeseke Looks good to merge.

**Pull request merged**